orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/util/PackageInfo.groovy

```diff
@@ -41,6 +44,8 @@ class PackageInfo {
     this.extractVersion = extractVersion
     this.mapper = mapper
     this.buildDetailExtractor = new BuildDetailExtractor()
+
+    filePattern = Pattern.compile("${stage?.context?.package}.*\\.${packageType}")
```

Comment:
I'd suggest fixing the tests if there aren't too many to align with what's expected when running in production.